apps/web/src/pages/Applications/ApplicationResumePage/ApplicationResumePage.tsx

```diff
@@ -51,15 +82,396 @@ export const getPageInfo: GetApplicationPageInfo = ({
+                defaultMessage: "Sort your experiences by",
+                id: "XuYilo",
+                description: "Label for experience sort dropdown",
+              })}
+              name="sortExperiencesBy"
+              options={[
+                {
+                  value: "date_desc",
+                  label: intl.formatMessage({
+                    defaultMessage: "Date (recent first)",
+                    id: "narnyP",
+                    description: "Sort experiences by date descending",
+                  }),
+                },
+                {
+                  value: "type_asc",
+                  label: intl.formatMessage({
+                    defaultMessage: "Type",
+                    id: "m7tUAm",
+                    description: "Sort experiences by type ascending",
+                  }),
+                },
+              ]}
+              hideOptional
+              trackUnsaved={false}
+            />
+
+            <Link
+              type="button"
+              mode="inline"
+              color="secondary"
+              href={paths.applicationResumeAdd(application.id)}
+            >
+              {intl.formatMessage({
+                defaultMessage: "Add a new experience",
+                id: "ON4+Yr",
+                description: "A link to add a new experience to your resume",
+              })}
+            </Link>
+          </div>
+          {hasSomeExperience ? (
+            <Accordion.Root type="multiple">
+              {experiences.map((experience) => {
+                return (
+                  <ExperienceAccordion
+                    key={experience.id}
+                    experience={experience}
+                    headingLevel="h3"
+                    editPath={paths.applicationResumeEdit(
+                      application.id,
+                      experience.id,
+                    )}
+                    showSkills={false}
+                  />
+                );
+              })}
+            </Accordion.Root>
+          ) : (
+            <Well>
+              <p data-h2-text-align="base(center)">
+                {intl.formatMessage({
+                  defaultMessage: "You don’t have any resume experiences yet.",
+                  id: "xuAFzV",
+                  description: "Null state messages for resume list",
+                })}
+              </p>
+            </Well>
+          )}
+          <Input
+            id="experienceCount"
+            name="experienceCount"
+            label=""
+            hideOptional
+            type="number"
+            hidden
+            rules={{
+              min: {
+                value: 1,
+                message: intl.formatMessage({
+                  defaultMessage: "Please add at least one experience.",
+                  id: "gyZV/3",
+                  description: "Error message if there are no experiences",
+                }),
+              },
+            }}
+          />
```

Comment:
This will need to updated to be of `type="hidden"` since this results in an empty label associated with a hidden input.

Something I have been doing to get a properly hidden input is:

```tsx
<input type="hidden" {...register("experienceCount")} />
```

Though, this poses a problem with the error message 😢

Reply:
Yeah, this was my initial approach based on your code but I abandoned it because it failed to display any error message.  :grimacing: I'll take another swing at it.

Reply:
OK, just hiding the label seems pretty easy.
[hide hidden input label](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6457/commits/5efc62699c14b4b109edfb2273fd53490f7014e9)